docs/docsite/rst/vmware/vmware_troubleshooting.rst

```diff
@@ -22,13 +22,87 @@ When debugging or creating a new issue, you will need information about your VMw
     $ export GOVC_URL=https://ESXI_OR_VCENTER_HOSTNAME:443
     $ govc find /
 
+Network settings with vmware_guest in ubuntu 18.04
+==================================================
 
-Troubleshooting Item 
+Setting the network with `vmware_guest` in ubuntu 18.04 is known to be broken, due to missing support for `netplan` in the `open-vm-tools`.
```

Comment:
These documents are reStructuredText (not Markdown). Inline `code-formatted quotes` need double backticks. You've got single backticks, which creates _italic text_.

Reply:
thx for the feedback. I will rework this.